surveys/2021-annual-survey/questions.md

```diff
@@ -649,6 +649,33 @@ Select one:
 > This question assess hiring sentiment. Although there is intrinsic uncertainty, it is easy to answer and forward looking.
 > It will also be interesting to see what the demand for Rust skills from companies is over time.
 
+### Rate how much the follow statements are reasons your team use Rust at work.
+
+Statements:
+
+- We use Rust for its performance characteristics.
+- We use Rust because we need to run software in a constrained or specialized environment.
+- Rust security and safety properties are important to us.
+- Rust allows us to build correct and bug free software.
+- We find it enjoyable or fun to program in Rust.
+- We already know Rust so it's our default choice.
+- We find it easy to prototype with.
+
```

Comment:
I think there should be an answer about expressivity of data types and the type system. Probably also a free form box for other major reasons

Reply:
Can you explain how you were differentiate this from the "relatively correct and bug free aspect". Typically expressivity is preferred for better modeling the domain and thus leading to more correct programs. Some might also find this enjoyable in which case this is covered by one of the other possible answers.